erpnext/stock/get_item_details.py

```diff
@@ -463,3 +467,10 @@ def get_default_bom(item_code=None):
 			return bom
 		else:
 			frappe.throw(_("No default BOM exists for Item {0}").format(item_code))
+			
+			
+def get_goss_profit(out):
+	out.update({
+		"gross_profit": ((out.price_list_rate - out.valuation_rate) * out.qty) * (out.conversio_rate or 1)
```

Comment:
spelling of conversion_rate?

gross_profit = (out.base_rate - out.valuation_rate) * out.qty

It is in company_currency